.gitignore

```diff
@@ -12,6 +12,7 @@
 data/*.txt
 data/*.ttf
 data/*.sfd
+/docs/
```

Comment:
Distributions prefer shipping docs with packages rather than only online. That's required some work in Julia, so if it works here please keep them.